<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="reviews_count">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>160</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QListWidget" name="TableList">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>55</y>
      <width>351</width>
      <height>201</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="View">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>350</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>View</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Pause">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>350</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="time">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>220</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>85</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Table List</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Run">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Save">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>350</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>400</y>
      <width>600</width>
      <height>30</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="Accept">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>270</y>
      <width>85</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
     </font>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>150</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>예상 수집 리뷰 수:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>220</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>예상 소요 시간:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>705</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
